crates/biome_js_type_info/tests/snapshots/infer_flattened_type_from_chained_invocation_of_promise_returning_function.snap

```diff
@@ -0,0 +1,121 @@
+---
+source: crates/biome_js_type_info/tests/utils/mod.rs
+expression: content
+snapshot_kind: text
+---
+```ts
+function returnsPromise(): Promise<number> {
+	return Promise.resolved(true);
+}
+
+returnsPromise().then(() => {});
+
+```
+
+```
+Object {
+  prototype: "Promise" members: {
+    required sync Method "all" {
```

Comment:
It looks like this is a static method, but the staticness isn’t indicated in the snapshot.

Also, not sure if this is a problem in this PR or if it was already the case: these methods seem to come from the prototype chain, so I think they make the snapshot unnecessarily noisy. If we can only show own properties, that would be even better.

Reply:
That's because the `Debug` implementation of `Class` doesn't print `members`. Was it on purpose or not @arendjr ?

Reply:
Yeah, that was on purpose indeed. If a class debug prints its members, and one of the members returns the class itself, such as `Promise.all() -> Promise`, you may get an infinite recursion :) And printing them is too noisy anyway. We may care about the members when printing the class at the top level (and even then, preferably only the "own properties"), but anywhere else we'd just care that it resolved to the right class.

Reply:
Ok, I suppose we'll have to somehow understand whether those members are inherited from the global prototype. I will hide them for now, thank you